sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateOperation.cs

```diff
@@ -13,8 +13,9 @@ namespace Azure.Security.KeyVault.Certificates
     /// </summary>
     public class CertificateOperation : Operation<KeyVaultCertificateWithPolicy>
```

Comment:
Unrelated. Should we rename this type to be less vague. Something like CreateCertificateOperation?

Reply:
Now that we don't return it from Cancel* or Move* methods of `CertificateClient`, we maybe could. The only issue is the type name would be different across languages. Some implement Cancel while others don't, but .NET is the only one to implement both Cancel and Delete.

Reply:
We discussed across languages and decided to keep it as-is, unless @schaabs has strong feelings about changing this.

Reply:
What would you do when you add another LRO to certificates?

Reply:
If we can't make `CertificateOperation` generic, we'd have to differentiate then. The main problem we discussed - while generally liking your suggestion conceptually - is that word consistency to go for. So if the type is `CreateCertificateOperation`, languages that have separate cancel and delete methods would/should look like `cancelCreateCertificateOperation` and `deleteCreateCertificateOperation`. It just looks...odd. But people agreed it wouldn't take much to make this change.

/cc @KrzysztofCwalina, @schaabs

Reply:
After much discussion here, offline, and with the other languages, we'll keep it as is. You make good points, but method names start getting long and looking weird, like `GetCreateCertificateOperation`, which all languages would have.